deploy/base/lookup-build-info-v0.1.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: tekton.dev/v1beta1
+kind: ClusterTask
```

Comment:
I know, this was why I decided to convert to pipelines with my other PR, then realized it was premature. It's really not that much work to change, but we do need to figure out how it will work with regards to namespaces/workspaces.